doc/source/user_guide/cookbook.rst

```diff
@@ -1336,21 +1336,27 @@ Values can be set to NaT using np.nan, similar to datetime
 Aliasing axis names
 -------------------
 
+.. versionchanged:: 1.1.0
```

Comment:
we should just remove this section entirely. it isn't really tested / nor well supported.